src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.cs

```diff
@@ -2253,7 +2253,19 @@ public class Mappings
     'New-AzNetworkInterfaceIpConfig': 'New-AzureRmNetworkInterfaceIpConfig',
     'New-AzExpressRouteCircuit': 'New-AzureRmExpressRouteCircuit',
     'Set-AzApplicationGatewayFrontendIPConfig': 'Set-AzureRmApplicationGatewayFrontendIPConfig',
-    'Get-AzInterfaceEndpoint': 'Set-AzureRmInterfaceEndpoint'
+    'Get-AzInterfaceEndpoint': 'Set-AzureRmInterfaceEndpoint',
```

Comment:
you need to regenerate this file, as it still references the cmdlets that are not part of the release

Reply:
Not sure how this is done, then, I replaced the whole networking module just by running the CreateAliasMapping.ps1 script. Are there extra steps which are necessary?